### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Inspect: "Role" and "State" provided for "Options" button is not correct.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#70]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS4.1.2

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Using Inspect Tool, Validate that Role and State provided for "Option" button is proper and correct.

**Actual Result:**
Following values are provided for the "Options" button:
Role= "grouping"
State= "normal"

**Expected Result:**
The role and state values for the "Options" button should be:
Role= "button"
State= "focusable, focused/not focused"

**User Impact:**
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachments:**
![bug 70_role and state is not correct for options button](https://user-images.githubusercontent.com/38521133/44572601-f7935000-a7a1-11e8-9881-2eb2977f414a.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by InduPriya1805** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS4.1.2; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
